infrastructure/networking.tf

```diff
@@ -49,6 +49,22 @@ resource "azurerm_virtual_network_peering" "tooling_to_bo" {
   provider = azurerm.tooling
 }
 
+resource "azurerm_virtual_network_peering" "bo_to_front_office" {
+  name                      = "${local.org}-peer-${local.service_name}-to-front_office-${var.environment}"
```

Comment:
For consistency, be great to change the _ to a -

Reply:
good spot!